js/header.js

```diff
@@ -10,14 +11,22 @@ function shrinkHeader() {
 }
 
 // Shrink header on mobile and resized window
-body.onload = function () {
+body.onload = () => {
     if (body.clientWidth <= headerShrinkWidth) {
         header.className = 'shrink';
     } else {
         header.className = '';
     }
+    if (window.innerWidth < 432)
+    {
```

Comment:
Braces on same line for consistency with rest of file

Reply:
done